x-pack/test/security_solution_api_integration/config/ess/config.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from './config.base';
+
+// eslint-disable-next-line import/no-default-export
+export default createTestConfig({
+  license: 'trial',
+  ssl: true,
+});
```

Comment:
I think this file should be called `config.base.trial.ts` and we should also have a `config.base.basic.ts`

Reply:
Good point!